frontend/src/wordInteractions.js

```diff
@@ -48,19 +50,22 @@ function loadWordsFromApi () {
   fetch('http://localhost:3000/word_banks')
     .then(resp => resp.json())
     .then(json => ALL_WORDS = json)
-    .then(chooseRandomWords);
+    .then(() => {
+      for (let i = 0; i < 3; i++) {
```

Comment:
maybe this for loop could be moved inside `chooseRandomWord()` and you could pass a number of words to grab as a parameter for that method? just a thought to make that method a bit more flexible

Reply:
Interesting thought. I think this comes down to game design. I did it this way because I figured there would only be one time, at the start of a game, where we want this to run multiple times in a row. We could also start with just one word instead of three, or I could build that into chooseRandomWords. That would be most valuable, though, if we ever want to do it multiple times again